#ALSA-29 implement item creation, change css style, add feedback message

AleksandarSavic95 opened the pull request:
- feedback messages tell if the create/delete operation succeeded,
- small css style changes for better UX :)

AleksandarSavic95 committed:
#ALSA-29 implement item creation, change css style, add feedback message

- feedback messages tell if the create/delete operation succeeded,
- small css style changes for better UX :)

calico-anne commented on src/components/TodoList.vue:
```diff
@@ -33,6 +44,14 @@ import { GET_ITEMS, STATUS, IS_LOGGED_IN } from '../constants'
 export default {
   name: 'TodoList',
+  props: [
+    'itemEvent', 'error'
```
Trebalo bi izbegavati samo nabrajanje `props`-a, bolje je ovako:
```
props: {
  itemEvent: String,
  error: String
}
```

( Ne znam sad koji su tipovi tacno )

calico-anne commented on src/components/ViewItem.vue:
```diff
@@ -99,7 +99,10 @@ export default {
-          this.$router.push({name: 'TodoList'})
+          this.$router.push({
+            name: 'TodoList',
+            params: { itemEvent: 'deleted' }
```
Mozda ove event-ove u konstante prebaciti. :)

AleksandarSavic95 committed:
#ALSA-29 refactor according to suggestions

- add types to `props`
- move `itemEvent` descriptions to `constants.js`
- simplify viewing `itemEvent` messages in `TodoList` component
